tests/Neo.UnitTests/Neo.UnitTests.csproj

```diff
@@ -12,6 +12,9 @@
   </ItemGroup>
 
   <ItemGroup>
+    <None Update="SmartContract\Manifest\TestFile\**">
```

Comment:
```suggestion
    <None Update="SmartContract\Manifest\Artifacts\**">
```

use artifacts folder?

Reply:
Artifacts is in .gitignore, I can't add files to it.